src/main/java/frc/robot/subsystems/snapback/hood/HoodIO.java

```diff
@@ -9,12 +10,15 @@ public interface HoodIO {
    * Inputs for Snapback's hood subsystem. Positions and velocities are of the output shaft, not the
    * motor shaft.
    */
+  @AutoLog
   public static class HoodIOInputs {
-    public Rotation2d position;
-    public double velocityRadiansPerSecond;
-    public double appliedVolts;
-    public double currentAmps;
-    public double temperatureCelsius;
+    public Rotation2d position = Rotation2d.fromRadians(0);
+    public double velocityRadiansPerSecond = 0;
+    public double appliedVolts = 0;
+    public double currentAmps = 0;
+    public double temperatureCelsius = 0;
+    public double positionSetpointRotationsPerSecond = 0;
+    public double positionErrorRotationsPerSecond = 0;
```

Comment:
The error here should be in units of position because we're trying to control its position, not its velocity. The actual unit is radians.